package.json

```diff
@@ -20,6 +20,7 @@
     "lint": "eslint --ignore-path .gitignore --ext .js,.ts,.tsx .",
     "stylelint": "stylelint --ignore-path .gitignore 'src/**/*.scss'",
     "test": "jest",
+    "typecheck": "tsc --noEmit",
```

Comment:
💯 

Does this actually mean that we can drop some stuff from ESLint?

Reply:
The typescript eslint plugin still has some useful checks in there (this is the list of default rules in the "recommended" config that we're using: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended.ts) so I would keep it